test/GovernorCountingFractional.t.sol

```diff
@@ -249,35 +264,37 @@ contract GovernorCountingFractionalTest is Test {
     return voteSplit.percentFor > 0 || voteSplit.percentAgainst > 0 || voteSplit.percentAbstain > 0;
   }
 
-  function _castVotes(Voter[4] memory voters, uint256 _proposalId) internal {
-    Voter memory voter;
-    for (uint8 _i = 0; _i < voters.length; _i++) {
-      voter = voters[_i];
-
-      assert(!governor.hasVoted(_proposalId, voter.addr));
+  function _castVotes(Voter memory _voter, uint256 _proposalId) internal {
+    if (_voter.weight == 0) return;
+    assertEq(governor.hasVoted(_proposalId, _voter.addr), false);
```

Comment:
Nit, you can use `assertFalse(governor.hasVoted(_proposalId, _voter.addr))` instead. Feel free to resolve this comment yourself.

Reply:
Thanks. Done in b202e74